CMakeLists.txt

```diff
@@ -90,9 +90,7 @@ if(APPLE)
     endif()
     if(NOT "${FreeOrionSDK_VERSION}" EQUAL "${FreeOrionSDK_INSTALL_VERSION}")
         file(REMOVE_RECURSE "${CMAKE_BINARY_DIR}/dep")
-        execute_process(COMMAND /usr/bin/hdiutil attach "${CMAKE_BINARY_DIR}/${FreeOrionSDK_FILE}")
-        execute_process(COMMAND ${CMAKE_COMMAND} -E tar xfj /Volumes/FreeOrionSDK/dep.tar.bz2 WORKING_DIRECTORY "${CMAKE_BINARY_DIR}")
-        execute_process(COMMAND /usr/bin/hdiutil detach "/Volumes/FreeOrionSDK")
+        execute_process(COMMAND ${CMAKE_COMMAND} -E tar xfj "${CMAKE_BINARY_DIR}/${FreeOrionSDK_FILE}" WORKING_DIRECTORY "${CMAKE_BINARY_DIR}")
```

Comment:
As I mentioned [here](https://github.com/freeorion/freeorion-sdk/pull/42#pullrequestreview-121768809) (commenting on PR freeorion/freeorion-sdk#42), I think this line needs to be adjusted (or a rename command added) to make sure the resulting folder containing the dependencies is named "dep" (and not something derived from the SDK archive's file name).